packages/tools/client-debugger/client-debugger-view/src/components/ContainerSelectionDropdown.tsx

```diff
@@ -0,0 +1,80 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { Dropdown, IDropdownOption, IDropdownStyles, IStackTokens, Stack } from "@fluentui/react";
+import React from "react";
+
+import { IFluidClientDebugger } from "@fluid-tools/client-debugger";
+import { HasClientDebuggers, HasContainerId } from "../CommonProps";
+
+/**
+ * {@link ContainerSelectionDropdownProps} input props.
+ *
+ * @internal
+ */
+export interface ContainerSelectionDropdownProps extends HasClientDebuggers, HasContainerId {
+    /**
+     * Take the selected container id to set as current viewed container id.
+     * @param containerId - current selected container id.
+     */
+    onChangeSelection(containerId: string): void;
+}
+
+/**
+ * A dropdown that displays all registered containers.
```

Comment:
I would recommend documenting this in terms of its use in selecting a container from a list. E.g. something like
```suggestion
 * A dropdown menu for selecting the Fluid Container to display debug information about.
```

Reply:
Thank you for suggestion. Replace the comments